eng/pipelines/runtime.yml

```diff
@@ -431,604 +431,604 @@ extends:
                 eq(dependencies.evaluate_paths.outputs['SetPathVars_non_mono_and_wasm.containsChange'], true),
                 eq(variables['isRollingBuild'], true))
 
-      # Build Mono AOT offset headers once, for consumption elsewhere
```

Comment:
All the commented code in this file needs to be reverted before merging. I'm avoiding running legs that won't get affected by this change to save some resources (mobile, mono, wasm).